manifests/init.pp

```diff
@@ -56,7 +56,9 @@
   $package_name   = $kafka::params::package_name,
   $package_ensure = $kafka::params::package_ensure,
   $group_id       = $kafka::params::group_id,
+  $system_group   = $kafka::params::system_group,
   $user_id        = $kafka::params::user_id,
+  $system_user    = $kafka::params::system_user,
```

Comment:
Can you add puppet4 datatypes for those two new params (and for the existing ones as well if you like)